### PR TITLE
Support flat config

### PR DESCRIPTION
**Pull request by OlivierZal**

### CODE REVIEW

**Comment by ljharb** [comment]
I'm not likely to drop support for the non-flat config for the better part of a decade, to be clear.

File: packages/eslint-config-airbnb-base/flat/rules/errors.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+module.exports = {
```
Comment (ljharb) [inline packages/eslint-config-airbnb-base/flat/rules/errors.js:1]: it’s definitely untenable to duplicate the config and make more than one source of truth.

Comment (OlivierZal) [reply packages/eslint-config-airbnb-base/flat/rules/errors.js]: It was still an incomplete draft but I give up on the direction I tried.

Comment (ljharb) [reply packages/eslint-config-airbnb-base/flat/rules/errors.js]: That's fine, but please reuse PRs instead of closing them and opening new ones.

**Pull request closed without merging**